include/range/v3/view/exclusive_scan.hpp

```diff
@@ -137,7 +137,7 @@ namespace ranges {
                         InputRange<Rng>,
                         MoveConstructible<T>,
```

Comment:
Should we change this constraint to `Semiregular<T>`? Since to hold the initial value in the view, we require the T to be a semiregular.

Reply:
I'd suggest changing to `CopyConstructible<T>` which will allow `semiregular_t<T>` to be `Copyable` as needed in `adaptor::begin` when you copy the initial value into the iterator. (Theoretically, you could leave the requirement at `MoveConstructible` and write the view such that it is a `ForwardView` when `T` is `CopyConstructible` and a move-only `InputView` otherwise, but that may be more work than you're interested in investing.)

Reply:
Thank you for the suggestion! I'll stick with `CopyConstructible<T>` for now.